tests/slow/test_sft_slow.py

```diff
@@ -106,7 +106,6 @@ def test_sft_trainer_transformers(self, model_name, packing):
 
             model = AutoModelForCausalLM.from_pretrained(model_name)
             tokenizer = AutoTokenizer.from_pretrained(model_name)
-            tokenizer.pad_token = tokenizer.eos_token if tokenizer.pad_token is None else tokenizer.pad_token
```

Comment:
we don't need this anymore

Reply:
I will do the same in our SFT script in `open-r1` once this is merged